### PR TITLE
added fetch by id and group endpints in /photos

### LINKED ISSUE #91: Adding endpoints to /photos
**Describe the feature**
Adding index and fetch by group endpoints on /photos route.

**Describe the solution you'd like** :zap:
- [ ]  Add search by index endpoint that allows user to fetch photos based on their index.
- [ ]  Add group endpoint to photos that allow the user to fetch a certain number of photos from a particular index.

**Are you working on this feature? (Yes)**

### PR DESCRIPTION
**Pull request by ritik307**
# Related Issue

-   closes #91 

# Proposed Changes

- [x]  Add search by index endpoint that allows user to fetch photos based on their index.
- [x]  Add group endpoint to photos that allow the user to fetch a certain number of photos from a particular index.

# Checklist

-   [x] Tests
-   [x] Translations

### CODE REVIEW

**Comment by ritik307** [comment]
@ShauryaAg @paramsiddharth plz review

File: src/api/routes/photos.ts [hunk 1]
```diff
@@ -16,5 +16,32 @@ router.get("/", (req: Request, res: Response, next: NextFunction) => {
         });
     }
 });
-
+router.get("/:index", (req: Request, res: Response, next: NextFunction) => {
```
Comment (ShauryaAg) [inline src/api/routes/photos.ts:19]: Let's change this `index` to `id`.

**Review by ShauryaAg — CHANGES_REQUESTED** [review]
Please properly lint the files before committing.

**Comment by ritik307** [comment]
@ShauryaAg Done!

**Comment by ShauryaAg** [comment]
@ritik307 Kindly lint the file properly.

**Comment by ritik307** [comment]
image

@ShauryaAg  when I run `npm run lint` it shows the following warnings which are not unusual because they've been shown on other routes too which I haven't made any changes to.
Kindly guide me on what to do.

**Comment by ShauryaAg** [comment]
@ritik307 You can ignore the warnings for now. Just make sure there are no errors.

**Comment by ritik307** [comment]
@ShauryaAg I have checked already there are no errors

**Comment by ShauryaAg** [comment]
Ok. @ritik307

**Review by ShauryaAg — APPROVED** [review]

**Pull request merged**